### PR TITLE
Ability to hide Info and Warning Messages as a User Setting

### PR DESCRIPTION
**Pull request by tyler-lemke**
- currently all or nothing, you hide all info, or all warning messages, or both
- hope to expand this later to allow the user to ignore specific types of info or warning messages, or even errors

### CODE REVIEW

File: src/test/suite/extension.test.ts [hunk 1]
```diff
@@ -85,4 +126,15 @@ suite('Extension Test Suite', () => {
...
+
+
+
+	/**
+	 * Test of isHiddenMessage()
+	 */
+	// test('isHiddenMessage()', async () => {
+	// 	const  result = IssueDiagnostic.isHiddenMessage(sampleWarningData.type);
+	// 	assert.strictEqual(result, true);
+	// });
```
Comment (celian-rib) [inline src/test/suite/extension.test.ts:138]: You should either uncomment or delete this test. :)

**Review by celian-rib — CHANGES_REQUESTED** [review]
Hi @tylerslemke, thanks for the PR, really appreciate it!

It's a very good addition to the extension and I'm sure it will be useful to many developers.

I will release a new version of the extension as soon as we merge this PR. 👍🏻

**Comment by celian-rib** [comment]
Nevermind I'll handle the cleanup :) thanks again !

**Comment by tyler-lemke** [comment]
Thanks. I shouldn't have let that sneak through!

**Pull request merged**